src/current/v23.1/create-and-configure-changefeeds.md

```diff
@@ -7,26 +7,45 @@ docs_area: stream_data
 - Many DDL queries (including [`TRUNCATE`](truncate.html), [`DROP TABLE`](drop-table.html), and queries that add a column family) will cause errors on a changefeed watching the affected tables. You will need to [start a new changefeed](create-changefeed.html#start-a-new-changefeed-where-another-ended). If a table is truncated that a changefeed with `on_error='pause'` is watching, you will also need to start a new changefeed. See change data capture [Known Limitations](change-data-capture-overview.html) for more detail.
 - Partial or intermittent sink unavailability may impact changefeed stability. If a sink is unavailable, messages can't send, which means that a changefeed's high-water mark timestamp is at risk of falling behind the cluster's [garbage collection window](configure-replication-zones.html#replication-zone-variables). Throughput and latency can be affected once the sink is available again. However, [ordering guarantees](changefeed-messages.html#ordering-guarantees) will still hold for as long as a changefeed [remains active](monitor-and-debug-changefeeds.html#monitor-a-changefeed).
 - When an [`IMPORT INTO`](import-into.html) statement is run, any current changefeed jobs targeting that table will fail.
 - {% include {{ page.version.version }}/cdc/virtual-computed-column-cdc.md %}
 
+### Recommendations for the number of target tables
+
 When creating a changefeed, it's important to consider the number of changefeeds versus the number of tables to include in a single changefeed:
 
 - Changefeeds each have their own memory overhead, so every running changefeed will increase total memory usage.
-- Creating a single changefeed that will watch hundreds of tables can affect the performance of a changefeed by introducing coupling, where the performance of a watched table affects the performance of the changefeed watching it. For example, any [schema change](changefeed-messages.html#schema-changes) on any of the tables will affect the entire changefeed's performance.
+- Creating a single changefeed that will watch hundreds of tables can affect the performance of a changefeed by introducing coupling, where the performance of a target table affects the performance of the changefeed watching it. For example, any [schema change](changefeed-messages.html#schema-changes) on any of the tables will affect the entire changefeed's performance.
 
 To watch multiple tables, we recommend creating a changefeed with a comma-separated list of tables. However, we do **not** recommend creating a single changefeed for watching hundreds of tables.
 
 {% include {{ page.version.version }}/cdc/recommendation-monitoring-pts.md %}
 
+#### System resources and running changefeeds
+
+Cockroach Labs recommends monitoring [CPU usage](ui-overload-dashboard.html) when you are running around 10 changefeeds on a cluster. A larger cluster will be able to run more changefeeds concurrently compared to a smaller cluster with more limited resources.
```

Comment:
would it be better to give a range of changefeeds instead of saying "around"? Does monitoring happen with 10+ or should happen before 10?